solutions/usaco-574.mdx

```diff
@@ -108,4 +108,38 @@ int main() {
 
 </CPPSection>
 
+<PySection>
+
+```py
+from collections import deque
+
+fullness, que = set(), deque()
+fullness.add((0, 0))
+que.append((0, 0))
+
+with open('feast.in') as r:
+	T, A, B = map(int, r.readline().split())
+
+while que:
```

Comment:
is this a dfs of sorts? add a quick comment explaining

Reply:
its bfs